docs/guides/plugin.md

```diff
@@ -91,6 +91,13 @@ fn main() {
 }
 ```
 
+## Conventions
+
+- Plugins should have a clear name with `vite-plugin-` prefix.
```

Comment:
vite masterrace 🙌

Reply:
noooo I didn't copy and paste why you asking